Jenkinsfile

```diff
@@ -490,6 +532,115 @@ az aks create                      \
+                                            // variables to reference a user in AWS with the correct privileges to create an EKS
+                                            // cluster: https://github.com/weaveworks/eksctl/issues/204#issuecomment-450280786
+                                            container('eksctl') {
+                                                sh "eksctl create cluster --name ${clusterName} --region ${awsRegion} --zones ${awsZones.join(',')} --version ${kversion} --node-type m5.large --nodes 3 --timeout 60m --kubeconfig \$KUBECONFIG --tags 'platform=${platform},branch=${BRANCH_NAME},build=${BUILD_URL}'"
+                                                waitForRollout("kube-system", 30)
+                                            }
+
+                                            writeFile([file: 'kubeprod-manifest.jsonnet', text: """
+(import "manifests/platforms/eks.jsonnet") {
+  config:: import "${clusterName}-autogen.json",
+  letsencrypt_environment: "staging",
+  prometheus+: import "tests/testdata/prometheus-crashloop-alerts.jsonnet",
+}
+"""
+                                            ])
+                                        }{
+                                            // update glue records in parent zone
+                                            def nameServers = []
+                                            container('aws') {
+                                                def output = sh(returnStdout: true, script: """
+id=\$(aws route53 list-hosted-zones-by-name --dns-name "${dnsZone}" --max-items 1 --query 'HostedZones[0].Id' --output text)
+aws route53 get-hosted-zone --id "\$id" --query DelegationSet.NameServers
+"""
+                                                )
+                                                for (ns in readJSON(text: output)) {
+                                                    nameServers << ns
+                                                }
+                                            }
+                                            insertGlueRecords(dnsPrefix, nameServers, "60", parentZone, parentZoneResourceGroup)
+                                        }
+                                    } finally {
+                                        // Delete EKS cluster
+                                        container('eksctl') {
+                                            sh "eksctl delete cluster --name ${clusterName} --timeout 60m"
+                                        }
+                                        container('aws') {
+                                            withEnv([
+                                                "PATH+JQ=${tool 'jq'}",
+                                                "HOME=${env.WORKSPACE}",
+                                            ]) {
+                                                sh """
+                                                    #!/bin/bash
```

Comment:
Add `set -xe`  (jenkins does this automatically with the default `sh` shell, but not if you provide your own #! line)

Reply:
Done.